client/src/Components/UpdateForm/index.js

```diff
@@ -20,19 +21,50 @@ class UpdateForm extends Component {
       description: details.description,
       name: details.name,
       prescription: details.prescription,
-      quantity: details.soldout
+      quantity: details.soldout,
+      medicineid: details.medicineid
     })
+    // const medicineid = details.medicineid
+    console.log("id", details.medicineid)
   }
 
   handleChange = event => {
     const { value, name } = event.target
     this.setState({ [name]: value })
+    // console.log("d", value)
   }
 
   toggle = event => {
     const { name } = event.target
     this.setState({ [name]: !this.state[name] })
   }
+  updateFormInfo = event => {
+    event.preventDefault()
+
+    const {
+      name,
+      company,
+      price,
+      description,
+      quantity,
+      prescription,
+      medicineid
+    } = this.state
+
+    axios
+      .patch("/api/pharmacy/medicine", {
```

Comment:
you are updating a specific medicine, you need the medicine name as a variable and add it to the endpoint url